### ISSUE TITLE
hello? troble

### ISSUE THREAD

**Issue by changhyeok-an** [ChristopherHX/osx-packaging-scripts#13]

**Comment by crownyzz** [comment]
yeah ive been having the same issue to, idk how to fix it maybe bcuz its new

**Comment by changhyeok-an** [comment]
> yeah ive been having the same issue to, idk how to fix it maybe bcuz its new

1.19.40is working but.. 1.19.5x high not working 56% still....

**Comment by crownyzz** [comment]
im going to keep trying closing and reopeing... will update you

**Comment by crownyzz** [comment]
just got news the new update doesn't work on mac, so if youre on a mac, well, sorry lol

**Comment by kcastroj** [comment]
The game works for me (Mac Pro Big Sur 11.6) it launches but the only problem is i can only see external servers, i don't see the featured servers

**Comment by FireTriangle132** [comment]
1.16.50+ doesn't work... It always stops at 56% I'm using macOS 10.13.6 (High Sierra)

**Comment by crownyzz** [comment]
I’m on macOS 12.3… will that work or do I have to downgrade?

**Comment by eisnerguy1** [comment]
I've been getting stuck with the same issue :( I've been able to get it to load maybe less than half the time with 1.19.50+ and when I do, I can't sign into my Microsoft Account. The sign in button does nothing. Going to try installing 1.19.41.01 next.

**Comment by swvza** [comment]
mines stuck at 56% lol

**Comment by Pysnake3** [comment]
Give this a try, it might work.

Mine kept getting stuck at 56% so I tried closing everything except the launcher. Didn't work.
Tried doing the opposite instead, with the launcher still open I opened Cinebench and started running a benchmark on all cores, went to the launcher and then clicked start. It didn't get stuck at 56% and I stopped playing to come and write this.
I'm not going to close the game to try and find out if I just got lucky or not.

Running osx 10.13.4 on a macbook pro mid 2012
Minecraft version 1.19.51

**Comment by uwukittycat** [comment]
Wow, pysnake it actually worked. Thx for the help.
